docs/06-contributors/999-rfcs/000-stories/story-15.md

```diff
@@ -0,0 +1,79 @@
+Initial feedback for TaskListApi Resource that usese cloud.Api 
+
+```ts (wing)
+
+bring cloud;
+
+interface ITaskListModel {
+  inflight get(id: str): str;
+  inflight add(title: str): str;
+  inflight remove(id: str);
+  inflight find(r: str): Set<str>;
+}
+
+resource TaskListModel implementes ITaskListModel {
+
+  init() {
+
+  }
+  inflight get(id: str): str {
+
+  }
+
+  inflight add(title: str): str {
+
+  }
+
+  inflight remove(id: str) {
+
+  }
+
+  inflight find(r: str): Set<str> { 
+
+  }
+}
+
+resource TaskListApi {
+  api: cloud.Api;
+  model: TaskListModel;
+  init(model: ITaskListModel) {
+    this.model = model;
+    this.api = new cloud.Api();
+    
+    this.api.post("/tasks", inflight (req: cloud.Api.ApiRequest): cloud.Api.ApiResponse => {
+      let title = Str.from_json(req.body.title);
+      let id = this.model.add(title);
+      return new cloud.Api.ApiResponse(status:202, body: Json.format(id));
+    });
+
+    this.api.get("/tasks/:id", inflight (req: cloud.Api.ApiRequest): cloud.Api.ApiResponse => {
+      let id = Str.from_json(req.params.id);
+      try {
+        let title = this.model.get(id);
+        return new cloud.Api.ApiResponse(status:202, body: Json.format(title));
+      } catch {
+        return new cloud.Api.ApiResponse(status:404);
```

Comment:
Why 404? Are we treating every exception as "not found"?

Reply:
I think this is good enough for now

Reply:
Do you disagree?